src/eduid_webapp/email/views.py

```diff
@@ -292,4 +293,22 @@ def resend_code(user, email):
     current_app.stats.count(name='email_resend_code', value=1)
 
     emails = {'emails': user.mail_addresses.to_list_of_dicts(), 'message': 'emails.code-sent'}
-    return emails
+    return EmailListPayload().dump(emails)
+
+
+@email_views.route('/get-code', methods=['GET'])
+@require_user
+def get_code(user: User):
+    """
+    Backdoor to get the verification code in the staging or dev environments
+    """
+    try:
+        if check_magic_cookie(current_app.config):
+            eppn = request.args.get('eppn')
+            email = request.args.get('email')
+            state = current_app.proofing_statedb.get_state_by_eppn_and_email(eppn, email)
+            return state.verification.verification_code
+    except Exception:
+        pass
```

Comment:
not sure what exceptions are caught here, but maybe log at least?

Reply:
Addressed - replying just to keep track :)